bedrock/externalpages/templates/externalpages/pocket/pocket-and-firefox.html

```diff
@@ -0,0 +1,102 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block pocket_css %}
+  {{ css_bundle('pocket-features') }}
+{% endblock  %}
+
+{% block page_title %}Pocket for Firefox{% endblock %}
+
+{% block content %}
+
+{% include 'externalpages/pocket/includes/nav.html' %}
+
+<section>
+{% call split(
+  block_class='mzp-l-split-center-on-sm-md',
+  media_class='mzp-l-split-h-center',
+  image_url='img/externalpages/pocket/save-inspirations.png',
+  )%}
+  <span class="tagline">Pocket for Firefox</span>
+  <h1 class="section-heading">Build your personal library of fascinating reads.</h1>
+  <p class="section-lede">Included inside Firefox, the <span class="pocket-logo-inline">Pocket Logo</span> button lets you save articles from across the web and read them in a quiet, private space.</p>
+  <div class="section-ctas">
+    {{ pocket_fxa_button(entrypoint='pocket', class_name='new-tab-pocket-cta', button_text='<span class="mzp-c-logo mzp-t-logo-sm mzp-t-product-family"></span>Activate Pocket in Firefox', optional_parameters={'s': 'fflearnmore', 'utm_campaign': 'landing-page', 'utm_content': 'page-button'}, optional_attributes={'data-cta-text': 'Activate Pocket', 'data-cta-type': 'activate pocket', 'data-cta-position': 'primary'}) }}
+    <a href="https://getpocket.com/signup">More ways to sign up</a>
+  </div>
+  {% endcall %}
+</section>
+
+<section class="explore-stories">
+  <div class="explore-stories-wrapper mzp-l-content mzp-t-content-md">
+    <h2 class="sub-section-heading">Get thought-provoking article recommendations</h2>
+    <p class="section-lede">In addition to saving stories from all over, you can also dig into the latest articles we're recommending in Pocket. Our editors find seriously interesting stuff — stories that dig deep into a subject, offer a new perspective, or make you think.</p>
+    <a href="https://getpocket.com/explore" class="discover-cta" role="button">Discover stories in Pocket</a>
+  </div>
+</section>
+
+<section>
+  {% call split(
+  block_class='mzp-l-split-center-on-sm-md',
+  media_class='mzp-l-split-h-center',
+  image_url='img/externalpages/pocket/reading-nook.png',
+  )%}
+  <h2 class="sub-section-heading">Your private reading nook</h2>
+  <p class="section-lede">When you’re ready to dig into what you’ve saved, Pocket provides a quiet, calm space that’s perfect for reading. It strips away all the distractions of the internet — like flashing banners and pop-ups — so you can really focus.</p>
+  {% endcall %}
+</section>
```

Comment:
**Praise**: i think good call not adding the weird floating text elements that the original page had